gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/JobStatistics.java

```diff
@@ -325,13 +338,26 @@ public Long totalBytesProcessed() {
       return totalBytesProcessed;
     }
 
+    /**
+     * Returns the query plan as a list of stages. Each stage involves a number of steps that read
+     * from data sources, perform a series of transformations on the input, and emit an output to a
+     * future stage (or the final result). The query plan is available for a completed query job and
+     * is retained for 7 days.
+     *
+     * @see <a href="https://cloud.google.com/bigquery/query-plan-explanation">Query Plan</a>
+     */
+    public List<QueryStage> queryPlan() {
+      return queryPlan;
```

Comment:
Should we return `null` when none is available and if so, should that documented?